src/controllers/rest/response/RESTResponse.ts

```diff
@@ -3,40 +3,53 @@ import RESTRequest from "../request";
 import { BloxyHttpError } from "../../../util/errors/errors";
 import { RESTResponseDataType } from "../../../interfaces/RESTInterfaces";
 
-
 export default class RESTResponse {
-    public controller: RESTController;
-    public request: RESTRequest;
-    public responseData: RESTResponseDataType;
-
-    // Public options: RESTResponseOptions;
-
-    constructor (controller: RESTController, request: RESTRequest, responseData: RESTResponseDataType) {
-        this.controller = controller;
-        this.request = request;
-        this.responseData = responseData;
-    }
-
-    async process (): Promise<RESTResponseDataType> {
-        const allProcessed = this.controller.responseHandlers.map(handler => handler(this));
-
-        if (allProcessed.every(processed => processed === true)) {
-            return this.responseData;
+  public controller: RESTController;
+  public request: RESTRequest;
+  public responseData: RESTResponseDataType;
+
+  // Public options: RESTResponseOptions;
+
+  constructor (
+    controller: RESTController,
+    request: RESTRequest,
+    responseData: RESTResponseDataType
+  ) {
+    this.controller = controller;
+    this.request = request;
+    this.responseData = responseData;
+  }
+
+  // eslint-disable-next-line require-await
+  async process (): Promise<RESTResponseDataType> {
+    const allProcessed = this.controller.responseHandlers.map((handler) =>
+      handler(this)
+    );
+
+    if (allProcessed.every((processed) => processed === true)) {
+      return this.responseData;
+    } else {
+      const error = allProcessed.find(
+        (err) =>
+          err instanceof BloxyHttpError
+          && err.name === "BloxyInvalidStatusMessageError"
```

Comment:
```suggestion
          && err.name === "BloxyHttpInvalidStatusMessageError"
```

Another one of the fixes on my fork